docs/tutorials/elasticdl_local.md

```diff
@@ -87,11 +86,23 @@ The following commands build the Docker image `elasticdl:mnist`. Please feel
 free to name it in any other name you like.
 
 ```bash
-cd model_zoo
+cd elasticdl/model_zoo
 elasticdl zoo init
 elasticdl zoo build --image=elasticdl:mnist .
 ```
 
+## Authorize the Job Execution
+
+If you are going to run ElasticDL job in Minikube for the first time. Please
```

Comment:
It is easier to explain RBAC using a URL than copy-n-paste the text.

This section should be part of the previous section of `The Kubernetes Cluster`.

Reply:
Good suggestion, moved to the section of `The Kubernetes Cluster.`